cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/internal/mazha/CouchClient.java

```diff
@@ -420,8 +419,7 @@ public void putAttachmentStream(String id, String rev, String attachmentName, St
             options.put("att_encoding_info", true);
         }
         options.put("open_revs", jsonHelper.toJson(revisions));
-        return this.getDocument(id, options, new TypeReference<List<OpenRevision>>() {
-        });
+        return this.getDocument(id, options,new CouchClientTypeReference<List<OpenRevision>>());
```

Comment:
needs a space after `options,`
and
we can probably benefit from having some static fields representing the types we use most e.g. :
`static final TYPE_REF_LIST_OPEN_REVS = new CouchClientTypeReference<List<OpenRevision>>();`
and equivalently for `List<Response>`, `DocumentRevs`